src/components/video.tsx

```diff
@@ -104,6 +106,22 @@ export const Video = (props: VideoProps) => {
     scrollCurrentVideo();
   }, [currentIndex]);
 
+  useEffect(() => {
+    // listening for shortcut keys
+    // Shift + n for next video and Shift + p for the prev video
+    window.addEventListener("keydown", event => {
```

Comment:
Rather than attaching a DOM level event listener, we just have a `handleKeyPress` method attached to `onKeyUp` prop for this component. Reference: https://stackoverflow.com/a/39559496/4396392

As a general good practice, whenever event listeners are added using `useEffect` hook, they can also be "cleaned up" like so: https://stackoverflow.com/a/55566585/4396392. This way, the event listener doesn't linger around after the component has unmounted.

Reply:
Sounds like a good idea
I tried keeping `onKeyUp` in the topmost `div`, but it was not working.
Any idea, what went wrong
This is basically the first line of `return`
```
<div className={queue.length !== 0 ? "video" : "video fullview"}  onKeyUp={handleKeyPress}>
```

The function handleKeyPress
```
let handleKeyPress = (e: KeyboardEvent) => {
    console.log("event is ")
  }
```

I have imported KeyBoardEvent from React
```
import React, { useState, useEffect, KeyboardEvent } from "react";
```

Reply:
Hey @rakaar! Sincere apologies for the late response. Didn't notice your replies. Can you push the changes you have attempted? We can try debugging with it.

Reply:
Pushed my code after modifications
it seems `onKeyUp` events works only in tags where input is allowed - `input`, `textarea`
So I used document.addEventListener, one for keyup, other for keydown
But thing is, i felt using a stateVariable was inevitable as we need to know somewhere that shift key was pressed
but i am changing the statevariable after key is released


But there is one issue-
it is not working the first time, it means u need to press shift twice for the first time
Reason - the first time ShiftKey is pressed the state variable doesn't change, i checked by console.log
Not sure why it happens - I think it is because [state variable updation is not instantaneous in react](https://stackoverflow.com/questions/41278385/setstate-doesnt-update-the-state-immediately)